content/en/serverless/installation/dotnet.md

```diff
@@ -13,32 +13,27 @@ further_reading:
   text: 'Submitting Custom Metrics from Serverless Applications'
 ---
 
-## Required setup
+## Prerequisites
 
-If not already configured:
-
-- Install the [AWS integration][1]. This allows Datadog to ingest Lambda metrics from AWS. 
-- Install the [Datadog Forwarder Lambda function][2], which is required to ingest AWS Lambda traces, enhanced metrics, custom metrics, and logs.
-
-After you have installed the [AWS integration][1] and the [Datadog Forwarder][2], follow these steps to instrument your application to send metrics, logs, and traces to Datadog.
+The [Datadog Forwarder Lambda function][1] is required to ingest AWS Lambda traces, enhanced metrics, custom metrics, and logs.
 
 ## Configuration
 
 ### Install
 
-1. Enable [AWS X-Ray active tracing][3] for your Lambda function.
-2. Install the [AWS X-Ray SDK for .NET][4].
+1. Enable [AWS X-Ray active tracing][2] for your Lambda function.
+2. Install the [AWS X-Ray SDK for .NET][3].
 
 ### Subscribe
```

Comment:
The Extension instructions would directly replace this section. The Extension for .NET is available in beta, so we should include it as a tab marking it as a beta (the default tab).